back-end/config.py

```diff
@@ -160,7 +173,7 @@ def generateConfig(
 
     try:
 
-        log("info", "Writing fresh configuration.")
+        log("debug", "Writing fresh configuration.")
```

Comment:
For clarification: When `LOG` is `True`, this message will currently not be logged since there is no function-"type" `debug` in the local custom function `log`. Is this the intended behaviour?

Reply:
Yes. Here are possible cases:

1. User declared `DEBUG=True` **--** All logs are PRINTED on console, even at the lowest Debug level.
2. `DEBUG=True` and also specified `LOG_DIR=/...` **--** All logs are STORED in specified logs dir, even at the lowest Debug Level.
3. `DEBUG=False` (which is also the same case as simply not assigning `DEBUG` env var at all) and NO `LOG_DIR` env **--**  All logs, and only the ones above Debug level, are PRINTED on console.
4. `DEBUG=False` and `LOG_DIR=/...` **--** All logs, and only the ones above Debug level, are STORED in specified Logs Dir.

Reply:
With the current implementation `DEBUG` is only considered at all if `LOG_DIR` is available.
Otherwise every message gets printed to the command line independent of whether `DEBUG` is `True` or `False`. So cases 1) and 3) mean that all messages including all debug messages always get printed to the command line.

For cases 2) and 4) the code in the custom `log` functions in `config.py` and `service.py` `logging.debug(message)` is currently absent. Maybe I am missing something, but with the current implementation I don't think any message with level `debug` gets logged to a file.

Reply:
Resolved and pushed.